### PR TITLE
cortex@362e902a5

### PR DESCRIPTION
**Pull request by edma2**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

cc @aallawala

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -90,18 +90,5 @@ replace github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-
...
-replace github.com/vimeo/galaxycache => github.com/thanos-community/galaxycache v0.0.0-20211122094458-3a32041a1f1e
-
-// v0.20.5 caused go mod checksum error
-replace github.com/go-openapi/spec => github.com/go-openapi/spec v0.20.6
-
-// the v6.0.5851 Prometheus depends on doesn't seem to exist anymore?
-replace github.com/ionos-cloud/sdk-go/v6 => github.com/ionos-cloud/sdk-go/v6 v6.0.4
-
-// attempt to fix Thanos' dep: github.com/efficientgo/tools/core@v0.0.0-20210129205121-421d0828c9a6: invalid version: unknown revision 421d0828c9a6
-replace github.com/efficientgo/tools/core => github.com/efficientgo/tools/core v0.0.0-20210326193628-425a09c04e05
```
Comment (aallawala) [inline go.mod]: These shouldn't be deleted

Comment (edma2) [reply go.mod]: Fixed. Not sure where that came from.

File: vendor/github.com/google/btree/btree.go [hunk 2]
```diff
@@ -796,7 +796,7 @@ func (t *BTree) DescendLessOrEqual(pivot Item, iterator ItemIterator) {
 }
 
 // DescendGreaterThan calls the iterator for every value in the tree within
-// the range [last, pivot), until iterator returns false.
+// the range (pivot, last], until iterator returns false.
```
Comment (aallawala) [inline vendor/github.com/google/btree/btree.go:799]: Since we're doing a go mod replace of prometheus, only prometheus changes in `vendor/` should appear.

**Comment by edma2** [comment]
All checks passed.

**Pull request closed without merging**